src/functional-test/kotlin/ft/FunctionalTest.kt

```diff
@@ -3,23 +3,33 @@ package ft
 import io.restassured.RestAssured
 import io.restassured.response.Response
 import org.assertj.core.api.Assertions.assertThat
-import org.junit.jupiter.api.Test
+import org.json.JSONObject
+import org.junit.jupiter.api.BeforeAll
 import org.springframework.http.HttpStatus
+import org.springframework.http.MediaType.APPLICATION_JSON_VALUE
+import java.util.Base64
 
-class FunctionalTest {
+open class FunctionalTest {
+  lateinit var lastResponse: Response
+  val token = "Bearer ${getToken()}"
+  val testCrn = "D006296"
 
-  private lateinit var lastResponse: Response
+  companion object {
+    val client_id = "bill-sclater"
+    val client_secret = "KbICZKm61'xNx+9y5Z63IvA-7(ohdu&Ubg0LF0m%\$q0suoF<P8S3E\$QV*;5h"
```

Comment:
Do you want to be hardcoding your secret here?

Reply:
Er.. probably not! Anyway, it's gone!